_stories/atoms/index.js

```diff
@@ -33,4 +34,5 @@ stories
     .add('TextArea', withReadme(...TextArea))
     .add('TextInput', withReadme(...TextInput))
     .add('Tooltip', withReadme(...Tooltip))
-    .add('Trend', withReadme(...Trend));
+    .add('Trend', withReadme(...Trend))
+    .add('ProgressBar', withReadme(...ProgressBar));
```

Comment:
super minor thing: this breaks the alphabetical order ⛏

Reply:
done